heron/scheduler-core/src/java/com/twitter/heron/scheduler/dryrun/DryRunRender.java

```diff
@@ -0,0 +1,32 @@
+// Copyright 2016 Twitter. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package com.twitter.heron.scheduler.dryrun;
+
+/**
+ * Interface of class that renders dry-run response
+ */
+public interface DryRunRender {
```

Comment:
Instead of one implementation that knows how to render both table and raw, you'd have two implementations, one for table and one for raw. You'd then bind each to their respective command line flags.